bramkouwen/Neatly-In-Motion
# Updating Products Page

## What does this PR do?
* It updates products.html 

## How do you check this PR?
* Pull down the branch
* Check the changed content in the products.html file  

## Screenshots 
None.
